kimchi/src/verifier.rs

```diff
@@ -104,6 +105,7 @@ where
         &self,
         index: &VerifierIndex<G>,
         public_comm: &PolyComm<G>,
+        public_input: &Vec<G::ScalarField>,
```

Comment:
I think clippy will yell at you here that it should be `&[G::ScalarField]`

Reply:
looks like it didn't, I'm confused o_O (usually it complains about that)

Reply:
Done in the follow-up PR